src/devSVG.cpp

```diff
@@ -306,6 +319,10 @@ void svg_line(double x1, double y1, double x2, double y2,
   write_style_end(svgd->file);
 
   fputs(" />\n", svgd->file);
+
+  fputs("</svg>\n", svgd->file);
```

Comment:
I think you might as well until that is merged. It's a simple enough idea that @yixuan might include it